tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+from src.Board import Board
+from src.InputParser import InputParser
+
+def makeBoardMoves(board: Board, moves):
+    parser = InputParser(board, True)
+    
+    for moveStr in moves:
+        move = parser.parse(moveStr)
+        parser.side = not parser.side
+        board.makeMove(move)
```

Comment:
Eventually i should've marked it a bit clearer. This should be converted to a fixture. `conftest.py` has a special meaning to `pytest` (it is kind of a setup file), it will be automatically **"run"** before the actual tests run.

Reply:
Ah, ok, sorry.